redis/Cargo.toml

```diff
@@ -71,7 +71,7 @@ default = ["acl", "streams", "geospatial", "script"]
 acl = []
 aio = ["bytes", "pin-project-lite", "futures-util", "futures-util/alloc", "futures-util/sink", "tokio/io-util", "tokio-util", "tokio-util/codec", "tokio/sync", "combine/tokio", "async-trait"]
 geospatial = []
-json = ["serde", "serde_json"]
+json = ["serde", "serde_json", "serde/derive"]
```

Comment:
my bad, i messed this one up the first time around

this is needed othersise the json tests dont seem to want to run at all

![image](https://user-images.githubusercontent.com/45675641/202289752-7f6b668c-0bfc-41e4-a812-538008d85cc6.png)

This is the error that this fixes

Reply:
adding `serde/derive` only imports some proc macros which don't end up in the compiled binary anyway, so this shouldn't cause issues